src/listener/Heading.php

```diff
@@ -27,10 +33,18 @@ public function process(Line $line)
 
     /**
      * {@inheritDoc}
+     * 
+     * @since 1.2.0 Added exception
```

Comment:
Since tag is not required in this case. If you want to point out a change in response you can do this in the exception throws phpdoc

```php
/*
 *@throws Since version 1.2.0 an exception is thrown when an uknown heading level is provided
 */
``` 

By definition its absolute valid what you are doing here, @since can exists multiple times with multiple infos, i just never saw this implementation for api doc renders, so its very common just to point out since when the property/method exists.

So i think we just should be consistent and therefore only add since when the method has introduced, but let me know your opinion. I am open to change my conventional behavior.

Reply:
Specifically for `@throws` I'm used to the format of `@throws [type] [<description>]`. Thus it would become something like `@throws Exception for unknown heading levels, since version 1.2.0`?

Another approach which is used for some tags is inlining them with `{}`, e.g. `@throws Exception for unknown heading levels {@since 1.2.0}`. I think this is not officially supported for `@since`, however, it is roughly the same as not using the tag and just writing it out, except it is way more readable.

Shall I use that?

```patch
-     * @since 1.2.0 Added exception
-     * @throws Exception for unknown heading levels
+     * @throws Exception for unknown heading levels {@since 1.2.0}
```

Reply:
> @throws Exception for unknown heading levels {@since 1.2.0}

perfect!